apps/docs/docs/learn/06-recipes/01-contexual-styles.mdx

```diff
@@ -0,0 +1,45 @@
+---
+# Copyright (c) Meta Platforms, Inc. and affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+sidebar_position: 5
```

Comment:
You need to update these numbers to order the recipes. I suggest putting `Variants` first as that is a common use-case.